src/test/java/jenkins/plugins/okhttp/api/ClientTest.java

```diff
@@ -164,6 +165,42 @@ public void testProxy() throws ExecutionException, InterruptedException, IOExcep
         }
     }
 
+    @Test
+    public void testProxyWithBasicAuthAndRealm() throws ExecutionException, InterruptedException, IOException {
+        jenkinsRule.jenkins.setProxy(new ProxyConfiguration("127.0.0.1", proxy.port(), "proxy-user", "proxy-pass"));
+        secureProxy("Basic realm=\"somerealm\"", "proxy-user:proxy-pass");
+        server.stubFor(WireMock.get("/hello").willReturn(aResponse().proxiedFrom(proxy.baseUrl())));
+
+        final OkHttpClient client = JenkinsOkHttpClient.newClientBuilder(new OkHttpClient())
+                .build();
```

Comment:
this `.build();` seems a bit lonely, can you attach it to previous line or maybe do the break line with `.newClientBuilder` and aligned them vertically ?

Reply:
Done